services/src/main/java/org/keycloak/authentication/authenticators/browser/AbstractUsernameFormAuthenticator.java

```diff
@@ -113,7 +127,7 @@ public boolean enabledUser(AuthenticationFlowContext context, UserModel user) {
         if (!user.isEnabled()) {
             context.getEvent().user(user);
             context.getEvent().error(Errors.USER_DISABLED);
-            Response challengeResponse = challenge(context, Messages.ACCOUNT_DISABLED);
+            Response challengeResponse = challenge(context, Messages.ACCOUNT_DISABLED, FIELD_USERNAME);
```

Comment:
Should this really be a field error? I mean a disabled account is not an input error, so why it should be assigned to a field?